src/Tizen.Data.Tdbc/Tizen.Data.Tdbc/Statement.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading.Tasks;
+
+namespace Tizen.Data.Tdbc
+{
+    /// <summary>
+    /// TDBC interface for representing statement.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public interface Statement : IDisposable
```

Comment:
In my opinion this interface could be merged into IConnection, or called sth like StatementExecutor,  SqlEngine, SqlSession.